### ISSUE TITLE
Add support for .md files

### ISSUE THREAD

**Issue by adowling2** [jckantor/nbpages#57]
In some cases, I would like to have a page as a markdown file instead of notebook. Examples: syllabus, text for best practices, chapter introduction, etc. For simple things, I find it a little cumbersome to file up a notebook.

One idea would be to support either .ipynb and .md files, e.g., nbpages is smart enough to handle 01-01-something.ipynb or 01-01-something.md. 

This is a summer enhancement. I also have not through through all of the implications of this request.

**Comment by jckantor** [comment]
Yes, this is something I’ve wanted to do for a while. It will require a daily thorough rewrite because of an assumption that all files are structured as JSON, which is how the nav bars and other data are inserted.  So a hack might be to rewrite .md files as .ipynb, then process.
